burn-book/src/saving-and-loading.md

```diff
@@ -0,0 +1,89 @@
+# Saving and Loading Models
+
+Saving your trained machine learning model to disk is quite easy, no matter the output format you choose. As mentioned in the [Record](./building-blocks/record.md) section, different formats are supported to serialize/deserialize models. By default, we use the `NamedMpkFileRecorder` which uses the [MessagePack](https://msgpack.org/) binary serialization format with the help of [smp_serde](https://docs.rs/rmp-serde/).
+
+```rust, ignore
+// Save model in MessagePack format with full precision
+let recorder = NamedMpkFileRecorder::<FullPrecisionSettings>::new();
+model
+  .save_file(model_path, &recorder)
+  .expect("Could not save model");
```

Comment:
"Could not save the model" => "Should be able to save the model".

Reply:
Oops, still not familiar with the conventions for this 😄